PWG/EMCAL/EMCALbase/AliMCParticleContainer.cxx

```diff
@@ -240,6 +240,8 @@ const char* AliMCParticleContainer::GetTitle() const
 {
   static TString trackString;
 
+  trackString = TString::Format("%s_pT%04d", GetArrayName().Data(), static_cast<int>(GetMinPt()*1000.0));
+  /*
   if (GetMinPt() == 0) {
```

Comment:
I would suggest just removing the old code - it's always in the history if needed

Reply:
Anyhow planned once the review is over.